packages/redux/src/recentlyViewed/actions/factories/fetchRecentlyViewedProductsFactory.ts

```diff
@@ -0,0 +1,60 @@
+import * as actionTypes from '../../actionTypes';
+import { areRecentlyViewedProductsFetched } from '../../selectors';
+import type { Dispatch } from 'redux';
+import type { FetchRecentlyViewedProductsAction } from '../../types';
+import type { FetchRecentlyViewedProductsFactory } from './types';
+import type {
+  GetRecentlyViewedProducts,
+  RecentlyViewedProducts,
+} from '@farfetch/blackout-client/recentlyViewed/types';
+import type { StoreState } from '../../../types';
+
+/**
+ * Method to create a thunk factory configured with the specified client fo fetch recently viewed products.
+ *
+ * @param getRecentlyViewedProducts - Get recently viewed products client.
+ *
+ * @returns Thunk factory.
+ */
+
+const fetchRecentlyViewedProducts: FetchRecentlyViewedProductsFactory<GetRecentlyViewedProducts> =
+
```

Comment:
Same as https://github.com/Farfetch/blackout/pull/113/files#r783181459

Reply:
In this case, I tried to remove but when format on saving, adds this line automatically.